agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -123,6 +123,58 @@ function collect_ara_data {
 	fi
 }
 
+function collect_stockpile_data {
+	# parse config
+	stockpile_path=$(getconf.py stockpile_path stockpile)
+	stockpile_user=$(getconf.py stockpile_user stockpile)
+	stockpile_local_remote_user=$(getconf.py local_remote_user stockpile)
+	stockpile_host_remote_user=$(getconf.py host_remote_user stockpile)
+	stockpile_output_path=$(getconf.py stockpile_output_path stockpile)
+	stockpile_dependencies=$(getconf.py -l stockpile_dependencies packages)
+
+	# install stockpile and its dependencies
+	for package in $stockpile_dependencies; do
+       		which $package &>/dev/null
+        	if [[ $? != 0 ]]; then
+                	debug_log "[$script_name] $package is not installed on the system, installing"
+                	echo "$package is not installed on the system, installing"
+                	yum install $package -y &>/dev/null
```

Comment:
For now, stockpile is just an ansible playbook.  Are there other functions of pbench-agent that rely on ansible?  If so, can we avoid installing it here, and just reporting to the user that without ansible installed these functions are not available?